src/transform/negate.js

```diff
@@ -2,8 +2,13 @@ var es = require("../es");
 
 function Negate(transform, node) {
     var expr = transform(node.expr);
-    var negate = es.Identifier(node.loc, "$negate");
-    return es.CallExpression(null, negate, [expr]);
+    if (typeof node.expr.data === "number") {
+        var literal = es.Literal(node.loc, node.expr.data);
+        return es.UnaryExpression(node.loc, true, "-", literal);
```

Comment:
The function `transform` should work here to change the `node.expr` into an ES AST node, so just use `var literal = transform(node.expr);`, please.